### ISSUE TITLE
Popovers should be buttons for better accessibility

### ISSUE THREAD

**Issue by jcoyne** [geoblacklight/geoblacklight#1548]
### Descriptive summary
Currently we're using a link without an href for the popovers, but this is not the supported markup in bootstrap https://getbootstrap.com/docs/5.3/components/popovers/#live-demo Accessibility experts also recommend using `role="button"` for this type of interacton.

We may want to consider building our own popover code that utilizes https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover rather than using Bootstrap 5's code which doesn't have these new accessibility features.

### Steps to reproduce the behavior

1. Do this
1. Then do this...

### Related work

Link to related tickets or prior related work here.

**Comment by thatbudakguy** [comment]
FWIW I think these are improvements, but the root of the problem is that this particular pattern of "little info icon that gives you more info, but doesn't do anything else" is kind of a minefield.

Tooltips are supposed to be extra information about a thing you can interact with, which is why they're supposed to target buttons, but [most people don't do that right](https://heydonworks.com/article/your-tooltips-are-bogus/). Clicking the info icon to get more info isn't "interaction" because the tooltip is supposed to explain what an existing button on the page does.

I think [this article on toggletips](https://inclusive-components.design/tooltips-toggletips/) explains it really well, and also makes the useful distinction that this pattern ("info icon you can click to get more info") is more properly called a toggletip, because it isn't related to any particular tool. The article also offers a couple of decent solutions, though all involve writing your own javascript to progressively enhance the toggletip.

A big part of the problem is that almost all implementations (including bootstrap's) don't result in the content of the toggletip being read to AT users at the time it is toggled. They might learn that something interactive is there, but even if they activate it, nothing happens until they "enter" the space created by the bubble. The second article's solutions use a dynamically populated `aria-live` region to fix this, so that at the moment the toggletip is triggered, the content is read aloud – just like how a sighted user would experience it immediately upon hovering/clicking.
